pages/api/table/[id].ts

```diff
@@ -0,0 +1,45 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import createDBClient from "@/common/client";
+
+export async function get(req: NextApiRequest, res: NextApiResponse) {
+  const dbURI = process.env.DATABASE_URL!;
+  const id = req.query.id as string;
+
+  if (!id) {
+    return res.status(400).json({ message: "Missing table name" });
+  }
+
+  const client = await createDBClient(dbURI);
+  if (!client) {
+    return res.status(500).json({ message: "Could not create DB client" });
+  }
+
+  try {
+    const result = await client.query(`SELECT * FROM ${id}`);
+    await client.end();
```

Comment:
client end should be in the finally block. currently if an error occurs the connections will leak